bundle/src/main/java/com/adobe/acs/commons/synth/SynthesizerServlet.java

```diff
@@ -0,0 +1,43 @@
+package com.adobe.acs.commons.synth;
```

Comment:
@dherges is this intended as a HTTP endpoint for Synthesizer? Or is this the "demo servlet"?

If its the demo, can we remove, if not can we move to the impl package?